brightwind/export/export.py

```diff
@@ -154,3 +154,97 @@ def export_csv(data, file_name=None, folder_path=None, **kwargs):
     else:
         raise NotADirectoryError("The destination folder doesn't seem to exist.")
     print('Export to csv successful.')
+
+
+def export_tws_file(eastings, northings, height, wspd_series, direction_series, wspd_std_series=None, site_name=None, file_name=None, folder_path=None):
+    """Export a WindSim timeseries tws file using a timeseries dataframe of wind speed and direction.
+
+    :param eastings: Eastings of the measurement location in meters.
+    :type eastings: float
+    :param northings: Northings of the measurement location in meters.
+    :type northings: float
+    :param height: Height that the timeseries represents in meters.
+    :type height: float
+    :param wspd_series: Series of wind speed variable
+    :type wspd_series: pandas.Series
+    :param direction_series: Series of wind directions between [0-360]
+    :type direction_series: pandas.Series
+    :param wspd_std_series: Series of wind speed standard deviations
+    :type wspd_std_series: pandas.Series
+    :param site_name: The site name to include in the file.
+           i.e 'Demo Data'
```

Comment:
The i.e. that you've included should align with the beginning of the sentence above (it's a bw library convention).

Reply:
And again below.

Reply:
Up[dated, however this is not the case in the export_tab_file?